pcg/docker/build.sh

```diff
@@ -36,7 +36,6 @@ cd "$SCRIPT_DIR" || exit
 if [ "$local_or_remote" == "remote" ]; then
     echo "Building for docker hub."
     docker buildx build --platform linux/amd64,linux/arm64 -t perfana/perfana-config-gen:$VERSION --push .
-    docker buildx imagetools create -t perfana/perfana-config-gen:latest perfana/perfana-config-gen:$VERSION
```

Comment:
can you merge latest main into this PR? it looks like this lined will be removed (and others below)